joomla/joomla-cms
# [4.0] Remove useless function to show all installation steps, if the entere…

## Summary of Changes
Removed useless function in installation javascript file, which showed all installation stepts if the database credentials are incorrect. 
![joomla-installation-pr](https://user-images.githubusercontent.com/20770106/64597344-d786fc80-d3b5-11e9-8891-878bdf7ed4fc.png)



### Testing Instructions
1. Start a joomla 4 installation
2. In the "databse configuration"-tab enter incorrect database credentials
3. Click on the "Setup Database Connection" button
4. Now you should get a view, similar to the screenshot above 
5. Now apply my patch
6. Retry the installation with incorrect database credentials
7. Now you should get a view where only the "Database Configuration"-Tab is shown


### Expected result
Stay on the Database Configuration"-Tab if i'am enter incorrect database-credentials.
![joomla-installation-pr_expected](https://user-images.githubusercontent.com/20770106/64597902-02be1b80-d3b7-11e9-8c0f-321397ea3be3.png)

### Actual result
All installation Tabs are shown. (See screenshot above)
